### ISSUE TITLE
Remove middle screen

### ISSUE THREAD

**Issue by wellemut** [kartevonmorgen/FairSync#29]
## Userstory
I want to click as little as possible to get my 500 imports into the map

## Problem
This screen only shows date and version number. All other Information I have on the next screen too
![Image](https://github.com/user-attachments/assets/58f1faad-0c93-4141-b888-79f96f6a227a)

## Solution

- [ ] Can we integrate the Date and Version-Number in the preview-field on the previeous screen? ![Image](https://github.com/user-attachments/assets/6ca6ec56-b3dc-4ea9-8b03-750ab7a7dd70)
- [ ] Can we integrate Date and Version-Numer with link in the next screen ![Image](https://github.com/user-attachments/assets/1ac6da5e-cdf1-4f20-a1d4-d8e4f57b3ca2)

- [ ] Can we move the destination-DB-selector to the settings-area?
- [ ] we could put a link in the header or footer, which directly leads you to the settings page (as a pop-up?) so you can allways access it in between.

**Comment by qknight** [comment]
No, the dialog was in extensive review weeks ago where you had the option to change it. This dialog holds so many implementation details like delayed loading of items and item selection for merging into. I don't see the point in days of work just to have a few clicks less on your side. Each dialog serves a dedicated purpose as discussed multiple times.

**Comment by wellemut** [comment]
Danke für das Feedback.
Wer Go with it

**Comment by wellemut** [comment]
@qknight das issue ist ja noch nicht gelöst. Daher hatte ich es wieder geöffnet, aber in prio b verschoben.

Nach wie vor bin ich der Meinung, wir brauchen noch zu viele Klicks pro Eintrag. Das ist ja ein Tool für Datenmassen. Wenn ich für 200 Einträge 2.000 Mal klicken muss oder 1.500 Mal ist das ein Unterschied.
Daher denke ich, dass das ihm Rahmen der nächsten Optimierungen mit dem nächsten Kunden relevant ist. "Low hänging fruit"...

Daher würden ich es gerne offen halten...

**Comment by qknight** [comment]
Ok, kannst du tracken, ggf. machen wir das dann spaeter. Ich nehme mich mal aus dem ticket raus, ich hab damit ja nichts direkt zu tun.
